prep-javascript-variables/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+var fullName = 'Jesus Martinez';
+console.log('Value of full name:', fullName);
+var numberOfPets = '4';
+console.log('Value of numberOfPets:', numberOfPets);
```

Comment:
Hi Jesus, you'll want to leave out the quotes so that the JavaScript engine sees `numberOfPets` as a number and not a string.

Reply:
done and done.